From 79d6221c3a4efebb45da136e7b0a4815bb8771df Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:12:29 -0400
Subject: [alsa] rme9652: fix a missing comma in channel_map_9636_ds

Message-id: <140972823573-14-patchset-jkysela@redhat.com>
Patchwork-id: 90714
O-Subject: [RHEL 7.1 PATCH 013/442] ALSA: rme9652: fix a missing comma in channel_map_9636_ds[]
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

The lack of comma leads to the wrong channel for an SPDIF channel.
Unfortunately this wasn't caught by compiler because it's still a
valid expression.

Reported-by: Alexander Aristov <aristov.alexander@gmail.com>
Cc: <stable@vger.kernel.org>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: 770bd4bf2e664939a9dacd3d26ec9ff7a3933210)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/rme9652/rme9652.c b/sound/pci/rme9652/rme9652.c
index b96d9e1..3717f2d 100644
--- a/sound/pci/rme9652/rme9652.c
+++ b/sound/pci/rme9652/rme9652.c
@@ -285,7 +285,7 @@ static char channel_map_9636_ds[26] = {
  /* ADAT channels are remapped */
  1, 3, 5, 7, 9, 11, 13, 15,
  /* channels 8 and 9 are S/PDIF */
- 24, 25
+ 24, 25,
  /* others don't exist */
  -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1
 };
-- 
1.7.1